requirements.txt

```diff
@@ -1,4 +1,4 @@
 sqlalchemy
 radicale<3.0.0
 mysqlclient==2.0.1
-pymssql<3.0
+pymssql==2.1.5
```

Comment:
what about `<=`?

In order to avoid raising errors with new dependencies check of pip?

Reply:
Done @moylop260